### PR TITLE
Updated the event listing view page filter.

### PR DESCRIPTION
**Pull request by kairamkondarajesh**
https://github.com/ymcatwincities/openy/issues/2438

Testing steps: 

- Log in as admin
- Visit an existing page, like /locations/south-ymca
- Edit the page and add both the Upcoming Events and Events Post Listing Paragraphs
-image
- Observe events with dates in the past DO NOT appear
- image
- Add a new event with an end date in the future, observe it DOES appear in both Paragraphs.

### CODE REVIEW

**Comment by bjwilkins** [comment]
Can one of the admins verify this patch? Use "o+k to test" or ''t+est this please" for manual build execution.

File: modules/openy_features/openy_prgf/modules/openy_prgf_event_listing/config/install/views.view.listing_event_posts.yml [hunk 1]
```diff
@@ -1,3 +1,4 @@
+uuid: 422c07a9-2351-4c2b-b37f-efc749a66368
```
Comment (podarok) [inline modules/openy_features/openy_prgf/modules/openy_prgf_event_listing/config/install/views.view.listing_event_posts.yml]: we should remove uuid from configs

File: modules/openy_features/openy_prgf/modules/openy_prgf_event_listing/config/install/views.view.listing_event_posts.yml [hunk 2]
```diff
@@ -12,14 +13,15 @@ dependencies:
     - user
     - verf
     - views_infinite_scroll
+_core:
+  default_config_hash: aWBfhdf0ZQ9f2R4OiQLWMmn8O_VSkLH8LCQqB-nj8uc
```
Comment (podarok) [inline modules/openy_features/openy_prgf/modules/openy_prgf_event_listing/config/install/views.view.listing_event_posts.yml]: Hash and core section must be removed

File: modules/openy_features/openy_prgf/modules/openy_prgf_event_listing/openy_prgf_event_listing.install [hunk 3]
```diff
@@ -29,3 +29,15 @@ function openy_prgf_event_listing_update_8002() {
...
+ * Update view listing event posts configs.
+ */
+function openy_prgf_event_listing_update_8003() {
+  $config_dir = drupal_get_path('module', 'openy_prgf_event_listing') . '/config/install/';
+  $config_importer = \Drupal::service('openy_upgrade_tool.importer');
+  $config_importer->setDirectory($config_dir);
+  $config_importer->importConfigs([
+    'views.view.listing_event_posts'
+  ]);
+}
```
Comment (podarok) [inline modules/openy_features/openy_prgf/modules/openy_prgf_event_listing/openy_prgf_event_listing.install]: Let's follow code standsrd at the EOF

**Review by podarok — CHANGES_REQUESTED** [review]
Small fixes needed, please

File: modules/openy_features/openy_prgf/modules/openy_prgf_event_listing/config/install/views.view.listing_event_posts.yml [hunk 4]
```diff
@@ -12,14 +13,15 @@ dependencies:
...
 id: listing_event_posts
 label: 'Listing Event Posts'
 module: views
 description: 'Latest news posts list for landing page.'
 tag: ''
 base_table: node_field_data
 base_field: nid
-core_version_requirement: ^8 || ^9
```
Comment (froboy) [inline modules/openy_features/openy_prgf/modules/openy_prgf_event_listing/config/install/views.view.listing_event_posts.yml:22]: Should this be removed @podarok?

**Review by froboy — CHANGES_REQUESTED** [review]
@kairamkondarajesh looks close. Please include the same changes in this module as well https://github.com/ymcatwincities/openy/blob/9.x-2.x/modules/openy_features/openy_prgf/modules/openy_prgf_event_latest/

File: modules/openy_features/openy_prgf/modules/openy_prgf_event_listing/config/install/views.view.listing_event_posts.yml (hunk 4 above)
Comment (podarok) [reply modules/openy_features/openy_prgf/modules/openy_prgf_event_listing/config/install/views.view.listing_event_posts.yml:22]: yes, all good here. It was a fail add during autoconfig fix phase for D8->D9 upgrade

**Comment by podarok** [comment]
ok to test

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build3154/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/3154/

**Comment by bjwilkins** [comment]
Open Y Upgrade Path site check installed at http://upgrade.openy.cibox.tools/build3154/

**Comment by sarah-halby** [comment]
@froboy please let me know when this is ready for QA.

**Comment by podarok** [comment]
retest this please

**Review by podarok — APPROVED** [review]
Looks good in terms of the code

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build3167/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/3167/

**Comment by bjwilkins** [comment]
Open Y Upgrade Path site check installed at http://upgrade.openy.cibox.tools/build3167/

**Comment by podarok** [comment]
retest this please

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build3169/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/3169/

**Comment by bjwilkins** [comment]
Open Y Upgrade Path site check installed at http://upgrade.openy.cibox.tools/build3169/

**Comment by froboy** [comment]
@sarah-halby this is good to QA

**Comment by sarah-halby** [comment]
Thanks, @froboy - I've let the QA team know. They will address when they return from the holidays. cc @podarok & @gianni-imagex

**Comment by AnastasiiaPys** [comment]
retest

**Comment by podarok** [comment]
retest this please

**Comment by podarok** [comment]
> retest

This CI server has different commands, @AnastasiiaPys

**Comment by hamrant** [comment]
@kairamkondarajesh please provide steps for review (we need this for QA)

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build3172/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/3172/

**Comment by bjwilkins** [comment]
Open Y Upgrade Path site check installed at http://upgrade.openy.cibox.tools/build3172/

**Comment by kairamkondarajesh** [comment]
> @kairamkondarajesh please provide steps for review (we need this for QA)



> @kairamkondarajesh please provide steps for review (we need this for QA)

Testing steps:
1. When Site owner/Admin users  hovering over the main navigation, the pencil icon should display. 
2. Moving your cursor to the pencil icon should keep it accessible. Clicking the pencil icon reveals a small menu, with "Quick Edit" among the choices.

**Comment by hamrant** [comment]
@kairamkondarajesh how these steps related to the issue description https://github.com/ymcatwincities/openy/issues/2438 ?

Based on `files changed` you included filter by `field_event_dates_end_value`, I think in steps should be something like this:
- Login as admin
- Edit event and set an end date to past time
- Go to page `/page-with-paragraph`
- Check that expired event not displayed

cc @froboy

**Review by hamrant — APPROVED** [review]

**Comment by sarah-halby** [comment]
@hamrant & @kairamkondarajesh - does the QA team have the steps to test they need? We still have this listed as in progress rather than ready for QA.

**Comment by froboy** [comment]
@kairamkondarajesh here's my suggestion for clear testing steps to elaborate on @hamrant's suggestion. Please update the issue description if you agree.

- Log in as admin
- Visit an existing page, like `/locations/south-ymca`
- Edit the page and add both the `Upcoming Events` and `Events Post Listing` Paragraphs
Edit_Branch_South_YMCA___Drush_Site-Install
- Observe events with dates in the past DO NOT appear
South_YMCA___Drush_Site-Install
- Add a new event with an end date in the future, observe it DOES appear in both Paragraphs.

**Comment by froboy** [comment]
@kairamkondarajesh thanks. 
@sarah-halby it should now be ready for testing.

**Comment by AnastasiiaPys** [comment]
retest this please

**Comment by podarok** [comment]
image
@AnastasiiaPys added you to whilelist of users for CI
Now your comments should work here
Please retry posting comment for retest this please for CI

**Comment by AnastasiiaPys** [comment]
retest this please

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build3173/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/3173/

**Comment by bjwilkins** [comment]
Open Y Upgrade Path site check installed at http://upgrade.openy.cibox.tools/build3173/

**Comment by AnastasiiaPys** [comment]
Tested. I added Upcoming Events and Events Post Listing Paragraphs to the /locations/east-ymca
And I can see events with dates  in the past are displaying 
image
Events post listing paragraph does not display anything
image
A new event with an end date in the future does not appear in both Paragraphs.
@froboy @kairamkondarajesh

**Comment by froboy** [comment]
@podarok it sounds like @AnastasiiaPys is saying this **failed** testing. Is it possible that the update hook schema numbers need to be updated to `9xxx` instead of `8xxx`?

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build3176/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/3176/

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build3175/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/3175/

**Comment by bjwilkins** [comment]
Open Y Upgrade Path site check installed at http://upgrade.openy.cibox.tools/build3175/

**Comment by bjwilkins** [comment]
Open Y Upgrade Path site check installed at http://upgrade.openy.cibox.tools/build3176/

**Comment by podarok** [comment]
> @podarok it sounds like @AnastasiiaPys is saying this **failed** testing. Is it possible that the update hook schema numbers need to be updated to `9xxx` instead of `8xxx`?

Seems like all is good on a build server @froboy 
image

image


image


Why concern?

**Comment by froboy** [comment]
> And I can see events with dates in the past are displaying
image

> A new event with an end date in the future does not appear in both Paragraphs.

@podarok these are both incorrect results. 

Expected:
- events with end date in the past DO NOT appear in either paragraph
- new event with end date in the future DOES appear in each paragraph.

**Comment by podarok** [comment]
> > And I can see events with dates in the past are displaying
> > image
> 
> > A new event with an end date in the future does not appear in both Paragraphs.
> 
> @podarok these are both incorrect results.
> 
> Expected:
> 
> * events with end date in the past DO NOT appear in either paragraph
> * new event with end date in the future DOES appear in each paragraph.

How do you see me to help you in this scope?

**Comment by froboy** [comment]
@podarok I only pinged you to ensure that the results were clear.

**Comment by froboy** [comment]
retest

**Comment by podarok** [comment]
retest this please

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build3187/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/3187/

**Comment by bjwilkins** [comment]
Open Y Upgrade Path site check installed at http://upgrade.openy.cibox.tools/build3187/

**Comment by podarok** [comment]
retest this please

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build3188/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/3188/

**Comment by bjwilkins** [comment]
Open Y Upgrade Path site check installed at http://upgrade.openy.cibox.tools/build3188/

**Comment by podarok** [comment]
@kairamkondarajesh @froboy any updates on this?

**Comment by froboy** [comment]
retest

**Comment by froboy** [comment]
retest this please

**Comment by podarok** [comment]
retest this please

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build3197/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/3197/

**Comment by bjwilkins** [comment]
Open Y Upgrade Path site check installed at http://upgrade.openy.cibox.tools/build3197/

**Comment by froboy** [comment]
I've gone through the testing steps and this looks good. http://upgrade.openy.cibox.tools/build3197/locations/south-ymca has the blocks I placed. @AnastasiiaPys you should be good to retest.

**Comment by AnastasiiaPys** [comment]
retest this please

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build3201/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/3201/

**Comment by bjwilkins** [comment]
Open Y Upgrade Path site check installed at http://upgrade.openy.cibox.tools/build3201/

**Comment by AnastasiiaPys** [comment]
Tested. Testing passed. Events with dates in the past are not displayed
image

**Comment by podarok** [comment]
Thank you.

**Comment by bjwilkins** [comment]
Can one of the admins verify this patch? Use "o+k to test" or ''t+est this please" for manual build execution.

**Pull request merged**